lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -98,7 +99,9 @@ class ReportUIFeatures {
 
     // Fireworks.
     const scoresAll100 = Object.values(report.categories).every(cat => cat.score === 1);
-    if (!this._dom.isDevTools() && scoresAll100) {
+    const hasAllCoreCategories =
```

Comment:
hm, I guess this would still get fireworks for custom config categories but if the user is going all that way to get some fireworks they probably deserve 'em 😆

Reply:
I kind of feel like it should be at least perf + any other categories need to be 100. So solo perf 100 would be ok.

But I also don't care that much and this seems fine :)

Reply:
solo + fireworks would require a refactor, since the fireworks were designed to work with the scores container header.